### ISSUE TITLE
There maybe a bug for REDS_dataset.py to remove testing clips

### ISSUE THREAD

**Issue by DengpanFu** [xinntao/EDVR#125]
In data preparation, original REDS validation set's clips were renamed starting from 240, but in [codes/data/REDS_dataset.py lin58](https://github.com/xinntao/EDVR/blob/master/codes/data/REDS_dataset.py#L57) 
`self.paths_GT = [
            v for v in self.paths_GT if v.split('_')[0] not in ['000', '011', '015', '020']
        ]`
['000', '011', '015', '020'] were filtered out for testing, but i think the real ones to filter out should be ['240', '251', '255', '260']. Am I understanding it wrong?@xinntao

**Comment by ckkelvinchan** [comment]
We take out the four clips in the training set (['000', '011', '015', '020']) as the test set. All videos in the validation set are used in training.
